.github/workflows/deploy.yml

```diff
@@ -131,9 +131,6 @@ jobs:
       - name: Delete all files
         run: rm -rf *
 
-      - name: Add CNAME for docs.theforeman.org
-        run: echo docs.theforeman.org > CNAME
```

Comment:
The alternative to this is to create `public/master/CNAME`. Of course you need to move this to below the `mkdir -p` statement.

Reply:
`public/CNAME`, without the master, but I think just having the file is cleaner?

Reply:
Yes, probably is.